src/rmkit/util/timer.h

```diff
@@ -0,0 +1,98 @@
+#ifndef RMKIT_TIMER_H
```

Comment:
wouldn't this file look so much better as cpy? 🤞🏿

Reply:
I'm happy to give it a shot! I wrote it in plain c++ just b/c I'm more familiar and I haven't put the time in to come up w/ a decent editor config for cpy (both ft=python and ft=cpp are close, but I haven't found a happy medium where commenting, indentation, and autocomplete do what I want them to). But I imagine the translation to cpy would be pretty straightforward in this case, so definitely don't mind if you prefer that most of rmkit is cpy 🙂

Reply:
commenting and indentation work for me. i don't have autocomplete in general (sorry), just the normal keyword completion. i set ft=cpy and have [cpy.vim](https://gist.github.com/raisjn/1dc8538b151a7e2a1e548128634b9288)

there's also ctags.d/ config in rmkit which you can use with upgraded ctags (universal ctags?) for tag gen / fast jumping between most functions and classes (but not all). also supports tagbar.vim (using context stack), but i don't remember if i had to make changes there. 

![Screenshot_2021-02-13_09-48-33](https://user-images.githubusercontent.com/70462544/107857011-a8618f80-6de0-11eb-9a9e-a7b4092b5685.png)